standard-hackathon-rules.md

```diff
@@ -10,14 +10,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 ## The rules of the competition
 
-1. There is no maximum or minimum team size. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
+1. There is no maximum or minimum team size, however we do recommending having up to 4 team members to maximize the hackathon experience. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
```

Comment:
I think this line is not grammatically correct

should be changed to `however we do recommend having up to 4 team members to maximize the hackathon experience`

Reply:
i agree!